packages/eas-cli/src/credentials/manager/SelectIosDistributionTypeGraphqlFromIosDistributionTypeEasConfig.ts

```diff
@@ -0,0 +1,65 @@
+import { IosDistributionType as IosDistributionTypeEasConfig } from '@expo/eas-json';
+import _ from 'lodash';
+
+import {
+  CommonIosAppCredentialsFragment,
+  IosDistributionType as IosDistributionTypeGraphql,
+} from '../../graphql/generated';
+import { promptAsync } from '../../prompts';
+import { Context } from '../context';
+
+export class SelectIosDistributionTypeGraphqlFromIosDistributionTypeEasConfig {
+  constructor(private iosDistributionTypeEasConfig: IosDistributionTypeEasConfig) {}
+  async runAsync(
+    ctx: Context,
+    iosAppCredentials: CommonIosAppCredentialsFragment | null
+  ): Promise<IosDistributionTypeGraphql> {
+    if (this.iosDistributionTypeEasConfig === 'store') {
+      return IosDistributionTypeGraphql.AppStore;
+    } else if (this.iosDistributionTypeEasConfig === 'simulator') {
+      return IosDistributionTypeGraphql.Development;
+    }
```

Comment:
This seems incorrect. You don't need any credentials to build an app for the simulator.